Updated UI to match color palette for issue

PanjaChaitanya opened the pull request:
I have updated the UI to match the existing color palette as mentioned in the issue. Let me know if any changes are needed.

AbhayPundora commented:
@PanjaChaitanya  thank you for this😊 , can u pls attach a ss of the preview, then i'll merge this PR

AbhayPundora commented:
actually i'm a BCA undergrad , so u are my senior , can you give me some advice for my future based on your experience

PanjaChaitanya commented:
> @PanjaChaitanya thank you for this😊 , can u pls attach a ss of the preview, then i'll merge this PR

Here’s the preview of the changes. Let me know if any modifications are needed! 😊
![Screenshot (465)](https://github.com/user-attachments/assets/83c09e0e-148c-4495-ac79-ea23a29e01a1)
![Screenshot (464)](https://github.com/user-attachments/assets/f780c301-2ea9-49fd-86b5-1dab86929656)

PanjaChaitanya commented:
> actually i'm a BCA undergrad , so u are my senior , can you give me some advice for my future based on your experience

Honestly I’m still learning and figuring things out myself, but based on what I’ve experienced so far...

🔹 Work on practical projects =>it helps more than just completing college assignments. Open-source contributions, like what we’re doing now, are a great way to gain real-world experience.

🔹 Strengthen your basics =>whether it’s web development, DSA, JS or any other field, having a solid foundation makes learning advanced topics easier.

🔹 Look for internships or freelance work =>even small projects add a lot to your resume. Also, some companies like J.P. Morgan and BCG offer virtual internships on Forage. They’re free and a great way to get hands-on experience and weightage to resume.

🔹 Build a portfolio and stay active on GitHub =>showcasing actual work makes a bigger impact than just listing skills on a resume. don’t wait until your final year to explore opportunities.

You’re already taking a great step by contributing here. Keep on moving ! 
And finally, if you don’t know something but are willing to learn—fake it till you make it! 🤪Confidence + effort goes a long way.😉

AbhayPundora commented:
💯 thanks ✌️

AbhayPundora commented:
@PanjaChaitanya hello, I separated the frontend and backend into client and api folders, but when I pushed them to GitHub, they got messed up. I don't know what to do.
Can you help me out?

PanjaChaitanya commented:
> @PanjaChaitanya hello, I separated the frontend and backend into client and api folders, but when I pushed them to GitHub, they got messed up. I don't know what to do. Can you help me out?

Yeah Sure brother, Can You give me the more details about it

AbhayPundora commented:
@PanjaChaitanya thank you, I first had everything in the client folder and pushed it to GitHub. Later, I removed the .git folder from it, created a root folder called high_book, and moved the client folder into it. I also created an api folder that currently only handles authentication. But when I pushed the high_book folder to GitHub, everything got messed up. The client folder doesn’t open properly on GitHub. Locally, the structure looks fine, though

PanjaChaitanya commented:
> @PanjaChaitanya thank you, I first had everything in the client folder and pushed it to GitHub. Later, I removed the .git folder from it, created a root folder called high_book, and moved the client folder into it. I also created an api folder that currently only handles authentication. But when I pushed the high_book folder to GitHub, everything got messed up. The client folder doesn’t open properly on GitHub. Locally, the structure looks fine, though

try removing that .git folder at the root and re initialize it ,then 'add .' ,commit, push  and why the ui issue still in open

AbhayPundora commented:
yes ! thank you ,  i closed the issue
